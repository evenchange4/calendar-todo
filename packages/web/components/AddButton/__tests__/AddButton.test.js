// @flow
import * as React from 'react';
import { shallow } from '@pisano/enzyme';
import toJson from 'enzyme-to-json';
import AddButton from '../AddButton';

it('should render AddButton', async () => {
  const wrapper = shallow(<AddButton />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
