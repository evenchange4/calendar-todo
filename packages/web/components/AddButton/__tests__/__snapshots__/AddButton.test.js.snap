// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AddButton 1`] = `
<React.Fragment>
  <Fixed>
    <WithStyles(Button)
      aria-label="Add"
      color="primary"
      onClick={[Function]}
      variant="fab"
    >
      <pure(Add) />
    </WithStyles(Button)>
  </Fixed>
  <EventForm
    onHide={[Function]}
    open={false}
  />
</React.Fragment>
`;
